docs/installation.md

```diff
@@ -218,19 +218,234 @@ On Linux, as an optional post-installation task, you may wish to setup the bot t
+
+https://docs.conda.io/en/latest/miniconda.html
+
+##### Verify file integrity with SHA256 checksum:
+
+official SHA256 checksum can be found here
+
+https://docs.conda.io/en/latest/miniconda_hashes.html
+
+Navigate to the directory with your file, and run SHA256 checksum of your file, check with following command
+
+```bash
+sha256sum ./Miniconda3-number-of-the-package
+```
+
+both checksum suppose to match
+
+##### Install conda:
+
+While in directory with your file, install conda
+
+```bash
+bash ./Miniconda3-number-of-the-package
+```
+
+Confirm with yes all questions
+
+After installation, it is mandatory to turn your terminal OFF and ON again.
+
+CTRL + D (close terminal), CTRL + ALT + T (open terminal)
+
+#### 2. Prepare Conda environment
 
-``` bash
-conda env create -f environment.yml
+
+enter the conda base environment:
+
+```bash
+conda activate base
+```
+
+If you want, you can prevent the (base) conda environment from being activated automatically as well.
+
+```bash
+conda config --set auto_activate_base false
+```
+
+
+##### Change the channels with upgrades:
+
+Conda as a package manager can download new packages from "channels". The best developed Conda channel, is not the default channel and is called `conda-forge`. The code below switches to it.
+
+```bash    
+# adding forge
+conda config --env --add channels conda-forge
+
+# make it strict
+conda config --env --set channel_priority strict 	
+```
+
+You can check the status of conda with the following code.
+
+```bash
+conda info
+conda config --show channels
+conda config --show channel_priority
+```
+
+#### 3. Freqtrade Conda Environment
+
+Now you have conda, but only base environment,
+
+```bash# download freqtrade
+conda env list
 ```
 
+It is time to setup environment of the Freqtrade itself:
+
+
+Conda `create` command, installs all nested dependencies, for the selected libraries automatically
+
+general structure of installation command is:
+
+```bash
+conda create -n [name of the environment] [python version] [packages]
+```
+
+so it can be
+
+```bash
+conda create -n freqtrade-conda python=3.8 pandas numpy ta-lib git wheel virtualenv
+```
+
+or if you expect, to use later jupiter for [data-analysis](data-analysis.md), use
+
+```bash
+conda create -n freqtrade-jupyter-conda python=3.8 pandas numpy ta-lib git wheel virtualenv jupyter
+```
+
+the same works for [docker](docker.md), for spyder and other useful programs.
+
+Further read on the topic:
+
+https://towardsdatascience.com/a-guide-to-conda-environments-bc6180fc533?gi=1db972389cd1
+
+https://metager.de/meta/meta.ger3?eingabe=ardsdatascience+guide+to+conda+environment
+
+!!! NOTE:
+It may happen that creating a new Conda environment, populated with selected packages at the moment of creation, takes less time than installing a large, heavy dependent package into the old environment,
+takes less time than installing a large, heavy dependent package into the old environment.
```

Comment:
Using Notes requires a very specific syntax, otherwise it'll fail to render

```suggestion
!!! Note
    It may happen that creating a new Conda environment, populated with selected packages at the moment of creation, takes less time than installing a large, heavy dependent package into the old environment, takes less time than installing a large, heavy dependent package into the old environment.
```

Reply:
Thanks, it is corrected now.